### PR TITLE
Find service area interactive

### PR DESCRIPTION
**Pull request by singhgag**

### CODE REVIEW

**Comment by dg0yal** [comment]
- [ ] Can we change the basemap, to say the gray canvas? otherwise all the samples start to look the same.
- [ ] Also I think the fill could do with a little more transparency to not overpower the basemap
- [ ] Can you change the `geometryAtOverlap` option to use `dissolve`? That'll make overlapping service areas appear nicer, otherwise they seem to clash with each other

**Comment by dg0yal** [comment]
everything else looks good

**Comment by dg0yal** [comment]
You need to create a PR for vnext. This on is for master.

**Comment by singhgag** [comment]
@dg0yal I have made the changes. Please review.

File: arcgis-ios-sdk-samples/Route & Directions/Find service area interactive/ServiceAreaSettingsVC.swift [hunk 1]
```diff
@@ -0,0 +1,63 @@
+//
+//  ServiceAreaSettingsVC.swift
```
Comment (dg0yal) [inline arcgis-ios-sdk-samples/Route & Directions/Find service area interactive/ServiceAreaSettingsVC.swift]: Change copyright

**Comment by singhgag** [comment]
@dg0yal Done.

**Pull request merged**